extensions/security/audittrail/applib/src/main/java/org/apache/causeway/extensions/audittrail/applib/dom/AuditTrailEntryRepositoryAbstract.java

```diff
@@ -60,12 +63,25 @@ public Class<E> getEntityClass() {
         return auditTrailEntryClass;
     }
 
+    @Override
     public AuditTrailEntry createFor(final EntityPropertyChange change) {
         E entry = factoryService.detachedEntity(auditTrailEntryClass);
         entry.init(change);
         return repositoryService.persistAndFlush(entry);
     }
 
+    @Override
+    public Can<AuditTrailEntry> createForBulk(Can<EntityPropertyChange> entityPropertyChanges) {
+        Collection<AuditTrailEntry> auditTrailEntries = repositoryService.execInBulk(() -> entityPropertyChanges.stream()
```

Comment:
If the definition of the supertype's `default` impl is the same as the callable, ie:

```
return entityPropertyChanges.stream()
                .map(change -> {
                    E entry = factoryService.detachedEntity(auditTrailEntryClass);
                    entry.init(change);
                    return repositoryService.persist(entry);
                }).collect(Collectors.toList());
```

then it would then allow this override to be simplified to something like:

```
return Can.ofCollection(
  repositoryService.execInBulk( () -> super.createForBulk(entityPropertyChanges) )
  );
```

Reply:
Hmmm….

Reply:
You can call the super of an implemented interface even when it’s a default.

But this will do just fine:
return Can.ofCollection(repositoryService.execInBulk(() -> entityPropertyChanges.map(this::createFor).toList()));